pkg/config/config.go

```diff
@@ -4,6 +4,12 @@ import "sort"
 
 type Config struct {
 	Migration int64
+	Auth      *Auth
+}
+
+type Auth struct {
+	Cert []byte
+	Key  []byte
```

Comment:
```suggestion
	Cert string
	Key  string
```

Storing certificates as strings even within MongoDB makes it easier for infrastructure engineers to troubleshoot certificate issues.